evalml/tests/component_tests/test_components.py

```diff
@@ -825,6 +826,59 @@ def test_all_estimators_check_fit(X_y_binary, test_estimator_needs_fitting_false
         component.feature_importance
 
 
+@pytest.mark.parametrize("data_type", ['li', 'np', 'pd', 'ww'])
+def test_all_transformers_check_fit_input_type(data_type, X_y_binary):
+    X, y = X_y_binary
+    X = pd.DataFrame(X)
+    y = pd.Series(y)
+
```

Comment:
Is it worth moving this to a common location, somehow?  It looks like bits and pieces of it are invoked a few times.

Reply:
Yep! I'll simplify this.